### ISSUE TITLE
A Plague Tale Requiem error

### ISSUE THREAD

**Issue by MaluNoPeleke** [Fr33dan/GPSaveConverter#24]
When I want to move my savegame from Game Pass to Steam for the game "A Plague Tale Requiem" I always get this error:
![image](https://user-images.githubusercontent.com/1446541/205456728-0ef934a6-d3df-4911-b9ae-b988460d2e1c.png)
This is how the window looks like:
![image](https://user-images.githubusercontent.com/1446541/205456757-395737c6-f002-41f2-a274-464a5bf1b50b.png)
Here is a TRACE log: 
[GPSaveConverter.log](https://github.com/Fr33dan/GPSaveConverter/files/10147134/GPSaveConverter.log)
How to fix it?

**Comment by Fr33dan** [comment]
Someone needs to create a file translation for this game. The issues tab here is full of requests for them but I haven't had the time to do so. They are documented at [here](https://github.com/Fr33dan/GPSaveConverter/wiki/File-Translations). This game it would not be too hard to adapt the example from Guardians of the Galaxy.

**Comment by MaluNoPeleke** [comment]
Thanks, I have read the documentation about it but it is unclear for what to do now. Should I create a file with different regex expressions and import it and what kind of format should it have?

**Comment by Fr33dan** [comment]
You've made me realized I've left a basic out of the documentation. You should use the view menu to show the file translations panel. Then you can type the various values in there and experiment to make sure when you select the file in one box, the corresponding file in the other is selected.

Having this files already will make it easier for users who have the game to work and create these translations than it is for me. I have to install the game create a fake directory structure with dummy files with the same names for the non-xbox version to test it and it's a big pain.

I've got an idea for an update to smooth the process, but haven't had time to implement it.

**Comment by MaluNoPeleke** [comment]
Okay, and by using that panel I can also move the files once I found out the right values? Or do I need to report them and wait for an update of your application?

**Comment by Fr33dan** [comment]
You will be able to copy them once the values in that panel are correct. All the database does is populate that panel, the contents of the panel determine what the software will (attempt to) do. This means you could break a good configuration accidentally which is why I hid it by default. If you report the working values here I can add it to the database so it will work for future users without them having to worry about it.

**Comment by MaluNoPeleke** [comment]
I have tried it and it should work accordingly to regex101 but it doesn't work: 
![image](https://user-images.githubusercontent.com/1446541/205512917-9e7cc1b9-3f23-441d-a50e-7e42d1e90233.png)

**Comment by Fr33dan** [comment]
I think the blob ID should just be fixed as "data" without and groups. You got that as I was typing :-)

Your non-Xbox files alls start with "/" which means they won't match. You need to add the slash to the non-xbox location so that it won't be in the file listing. And I am going to create an issue to fix later to do it automatically as part of input validation.

**Comment by MaluNoPeleke** [comment]
The game complains that all the files are corrupted and cannot be used. Is that a game specific issue or a tool specific one?

**Comment by Fr33dan** [comment]
So which direction are you trying to go? non-Xbox -> Xbox or the other way? Unless I've forgotten something that error occurs when you are trying to move a non-Xbox file -> Xbox, but there are more than one possible match on the containers regex. Your screenshots show non-Xbox file created which would mean you did a Xbox -> non-Xbox move.

I would try clearing out your WGS folder, running the game to let Xbox re-build it from the cloud then try again moving only the slot00 file. Check the edit date before and after you copy and to make sure the right file has been copied. If you are sure copied the right file then it is some thing in the game they are doing different between the versions.

**Comment by MaluNoPeleke** [comment]
Yes, I copied from Xbox to non Xbox and the file size differs quite a bit (few kb Steam default save game vs. nearly 10mb Xbox my late game save).
I renamed it correctly as I didn't know where exactly I should change something in your tool regarding the slash /

**Comment by Fr33dan** [comment]
I am refering to "File > Edit non-Xbox file location". If you didn't use that function how did your non-xbox save file get populated? In my testing using the "Select non-Xbox location" includes this slash. 

I had not seen that fork it looks like he fixed a feature I wrote but hadn't fully tested and thus wasn't part of the release version (and if based on a quick look at his changes it may have been bugged). That and some bug fixes that fix issues I haven't experienced or seen reported, but are good changes. Well except removing some whitespace that I think improved readability but that's more preference than anything else. I may merge them next time I get a chance to actually work on this beyond making some comments here.

Things are settling down now that I'm finishing up a transition into a new job so there may be some movement finally before too long.
